api/src/pcapi/utils/crypto.py

```diff
@@ -20,13 +23,28 @@ def _hash_password_with_md5(clear_text: str) -> bytes:
     return md5(clear_text.encode("utf-8")).hexdigest().encode("utf-8")
 
 
+def _hash_password_with_sha3_512(clear_text: str, salt: bytes) -> bytes:
+    """Hash a password using SHA3-512. output format: sha3_512$salt$hashed_password"""
+    if not salt:
+        salt = os.urandom(16).hex().encode("utf-8")
+    hashed_password = sha3_512(clear_text.encode("utf-8") + salt).hexdigest().encode("utf-8")
+    return b"$".join(
+        [b"", b"sha3_512", salt, hashed_password],
+    )
```

Comment:
[micro-détail / style] En incluant un "$" dans `"$sha3_512"`, il est encore plus clair que le résultat final commence par un "$" : 

```suggestion
    return b"$".join([b"$sha3_512", salt, hashed_password])
```

(Au passage, pas besoin de "," après l'argument de `join`. Tout tient largement en une ligne, et la virgule force Black à passer l'argument  à la ligne.)